src/workbook/stakeholder-sheet/holdings-columns.ts

```diff
@@ -57,6 +60,8 @@ export class StakeholderGroupColumn {
       .addCell("Stakeholder Group", Styles.subheader)
       .addBlankCells(stakeholders.length, Styles.default)
       .addBlankCell(Styles.default)
+      .addBlankCell(Styles.default)
+      .addBlankCell(Styles.default)
```

Comment:
**Non-blocking:** Use `.addBlankCells` instead of multiple `.addBlankCell` calls.